lib/devise/test/controller_helpers.rb

```diff
@@ -139,7 +139,7 @@ def _process_unauthenticated(env, options = {})
 
           status, headers, response = Devise.warden_config[:failure_app].call(env).to_a
           @controller.response.headers.merge!(headers)
-          @controller.response.content_type = headers["Content-Type"] unless Rails.version.start_with?('5')
+          @controller.response.content_type = headers["Content-Type"] unless Rails.version.start_with?('5', '6.0')
```

Comment:
```suggestion
          @controller.response.content_type = headers["Content-Type"] unless Rails::VERSION::MAJOR >= 5
```

Reply:
This is better because we won't need to change it in future major versions again.

Reply:
@tegon Yeah totally agree.

as a related point, I'd be happy to add a separate PR to add rails 6 to the travis config, so we can catch these things earlier.

Was expecting 6.0.0.beta1 to be tagged yesterday,
but seems its been delayed a little.

Reply:
@tegon still waiting on the rails beta1 release,
so will wait and see.